### DISCUSSION TITLE
Please add Yugabyte to the list of supported databases

### DISCUSSION THREAD

**Discussion by udf2457** [directus/directus#15360] [Feature Requests]
At present, CockroachDB is the only multi-master [listed as supported](https://docs.directus.io/self-hosted/installation/cli.html).

The problem with this is that sometime in the last few years, CockroachDB [transitioned to a hybrid-license model ](https://www.cockroachlabs.com/docs/stable/licensing-faqs.html)where the latest version starts off as "BSL" license before eventually transitioning to Apache some time later.

Meanwhile, Yugabyte have a [firm commitment to 100% Apache licensing.](https://www.yugabyte.com/blog/why-we-changed-yugabyte-db-licensing-to-100-open-source/)

It would be nice if Directus formally supported Yugabyte.

**Comment by azrikahar** [comment]
@FranckPachot the Developer Advocate at Yugabyte did share a great write up on it over here: https://dev.to/yugabyte/directus-on-yugabytedb-4fpl 👍

As described the article, it's definitely possible, but not as out-of-the-box yet. We can track the in-progress items over at https://github.com/yugabyte/yugabyte-db/issues/1124.

---

Discord message link for additional reference: https://discord.com/channels/725371605378924594/793141711278833676/959483851061747753

**Comment by udf2457** [comment]
Thank you @azrikahar I've subscribed to notifications on that Github !

**Comment by barakplasma** [comment]
I tried using https://github.com/citusdata/citus as another distributed/multi-node Postgres-based database and it seems to be working.
Tried it with this https://github.com/citusdata/docker/blob/master/docker-compose.yml and 
`docker run   -p 8055:8055   -e KEY=255d861b-5ea1-5996-9aa3-922530ec40b1   -e SECRET=6116487b-cda1-52c2-b5b5-c8022c45e263 -e DB_CLIENT=pg -e DB_USER=postgres -e DB_HOST=172.17.0.1 -e DB_PORT=5432 -e DB_DATABASE=directus directus/directus`

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
